### PR TITLE
Adding a file for fuquay varina

### PR DESCRIPTION
**Pull request by githubteacher**
This will add a file about GitHubTeacher's hometown. What would you like to know?

### CODE REVIEW

File: places/githubteacher-fuquay.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Fuquay-Varina, NC
+
+### Places to Eat
+
+### Fun things to do
+
```
Comment (brianrudy055) [inline places/githubteacher-fuquay.md]: Are there any breweries in town that offer tours or tastings?

File: places/githubteacher-fuquay.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+### Awesome places to see
```
Comment (brianrudy055) [inline places/githubteacher-fuquay.md]: Are there any places near a beach and/or the ocean?

**Comment by brianrudy055** [comment]
@githubteacher  NC is one of the places I want to vacation to so any travel tips would be fantastic!  Thanks!

**Comment by githubteacher** [comment]
I think this looks close - @gbarry1 can you take a look and let me know if it looks ok to :ship:?

**Pull request merged**